torch/onnx/symbolic_opset11.py

```diff
@@ -177,7 +177,7 @@ def index_put(g, self, indices_list_value, values, accumulate=False):
     rank = sym_help._get_tensor_rank(values)
     if rank is not None and rank == 0:
         values = expand(g, values, values_shape, None)
-    values = g.op("Reshape", values, values_shape)
+    values = sym_help._reshape_helper(g, values, values_shape, allowzero=0)
```

Comment:
nit: Should we add an underscore in the middle(allowzero -> allow_zero) to follow the same naming rule as others?

Reply:
I think allowzero should be fine as allowzero is the attr name used by ONNX spec. Also I was wondering which naming rule you were referring to?